File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        vector<int> freq(26, 0);
        for(int i=0;i<t.size();i++) freq[t[i]-'a']++;
        for(int i=0;i<s.size();i++) freq[s[i]-'a']--;
        int res=0;
        for(int i=0;i<26;i++) res+=max(0, freq[i]);
        return res;
    }
};

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
       int res = 0, freq[] = new int[26];

    for(char c : s.toCharArray())
        ++freq[c - 'a'];

    for(char c : t.toCharArray())
        if(freq[c - 'a'] > 0)
            --freq[c - 'a'];
        else
            res++;

    return res;  
    }
}